api/labels.py

```diff
@@ -0,0 +1,30 @@
+from maxfw.core import MAX_API, CustomMAXAPI
+from flask_restplus import fields
+from core.model import ModelWrapper
+
+model_label = MAX_API.model('ModelLabel', {
+    'id': fields.String(required=True, description='Class label identifier'),
+    'name': fields.String(required=True, description='Class label'),
+})
+
+labels_response = MAX_API.model('LabelsResponse', {
+    'count': fields.Integer(required=True,
+                            description='Number of class labels returned'),
+    'labels': fields.List(fields.Nested(model_label),
+                          description='Class labels that can be predicted by '
+                                      'the model')
+})
+
+
+class ModelLabelsAPI(CustomMAXAPI):
+
+    model_wrapper = ModelWrapper()
```

Comment:
The issue I have with this approach is that this instantiates 2 `ModelWrappers` - this is a waste of resources since we're loading the model graph & weights into memory just for the sake of getting the labels.

While it's ideal to separate out `ModelLabelsAPI` into `labels.py`, in other models I have just put the labels API into `predict.py` because of this double wrapper issue...

see e.g. https://github.com/IBM/MAX-Named-Entity-Tagger/blob/master/api/predict.py#L45

I am not really sure of a good workaround that would enable achieving both goals.

Reply:
That makes sense to me, that was how I had done it before. I'll make the updates

Reply:
updated